content/posts/2019-02-14--huodong-c1/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: C位之战，邀你开声
+title: 三校联合剧本杀等你来
```

Comment:
I might be wrong here, but I feel this diff might be made by accident since we have [https://github.com/NUCSSA/nucssa-website/pull/24/files#diff-be6f69b9ee5cff2c953dfa648ede64ba](https://github.com/NUCSSA/nucssa-website/pull/24/files#diff-be6f69b9ee5cff2c953dfa648ede64ba)

Reply:
thanks, will change it